### DISCUSSION TITLE
Labels and announcements category now available 🚀

### DISCUSSION THREAD

**Discussion by evi-liu** [community/community#3979] [Discussions]
Thanks to all of your helpful feedback, we launched two highly requested features today: **Labels** and the **Announcements category format**.

## Labels
Labels will help maintainers triage and organize their discussions. You'll see them now available on the sidebar of every discussion.

![image](https://user-images.githubusercontent.com/8809849/118721102-48afc180-b7df-11eb-8c38-f13bff687487.png)

## Announcements category

When creating a new custom category, maintainers may now select the Announcements format where only maintainers and admins can post discussions to these categories.

![image](https://user-images.githubusercontent.com/8809849/118721293-814f9b00-b7df-11eb-9b5d-5d0205dec14b.png)

Feel free to give both these new features a try and let us know what you think!

**Comment by pointlessone** [comment]
Is there a way to subscribe only to Announcements?

I found a way to subscribe to all discussions but this might be a bit too noisy if people want to only get notifications for announcements.

**Reply by huehnerlady** [reply]
I agree this would be a great feature. not JUST to announcements, but to specific categories in general, see and vote [here](https://github.com/github/feedback/discussions/3951) 😃

**Reply by Erin-Cecele** [reply]
I agree this would be a really helpful feature! It also looks like this idea is related to #3951.

**Reply by ZokRZHON** [reply]
да действительно полезн связнь

**Comment by ngajhede** [comment]
Fantastic! 🎉 Labels are very useful to us.

**Reply by ZokRZHON** [reply]
правилно говориш пригадица

**Comment by Erin-Cecele** [comment]
@evi-liu Very excited about labels. Thank you for adding this feature. Question: is there a way for Discussions labels to differ from Issues labels? Or do they have to be the same categories?

**Reply by evi-liu** [reply]
They're currently the shared label sets, but we'd love more feedback on this. What value would you get out of differentiating the label sets?

**Reply by Erin-Cecele** [reply]
We use our issues board to communicate/bring issues forward to our developers, so the topics would be very different. Issues would skew more technical and code-related, whereas discussions are more user-oriented and based on the front-end experience of our publishing tool. 

For example, our issue labels include bug, good first issue, invalid, and dev (tbh we're still developing our full label list). Whereas for discussions, people frequently have questions about things like installation or the deployment process. Labels, like install or deploy, would make wading through our Get Help section so much easier.

We're thrilled about the new ability to turn a discussion into an issue, but we still want to make sure people are posting in the appropriate places. Shared labels might contribute to blurred lines.

I hope this is helpful!

**Reply by ZokRZHON** [reply]
Да вамь поможет

**Comment by Br0w07** [comment]
Update for new system version

**Comment by BankEang** [comment]
> Thanks to all of your helpful feedback, we launched two highly requested features today: **Labels** and the **Announcements category format**.
> 
> ## Labels
> Labels will help maintainers triage and organize their discussions. You'll see them now available on the sidebar of every discussion.
> 
> ![image](https://user-images.githubusercontent.com/8809849/118721102-48afc180-b7df-11eb-8c38-f13bff687487.png)
> 
> ## Announcements category
> When creating a new custom category, maintainers may now select the Announcements format where only maintainers and admins can post discussions to these categories.
> 
> ![image](https://user-images.githubusercontent.com/8809849/118721293-814f9b00-b7df-11eb-9b5d-5d0205dec14b.png)
> 
> Feel free to give both these new features a try and let us know what you think!

**Comment by prudhomm** [comment]
We can add a new category Announcement 
![Screenshot 2021-05-30 at 11 32 21](https://user-images.githubusercontent.com/151693/120099221-b55d8100-c13a-11eb-80c9-51a499011ada.png)

but when starting an Announcement discussion is not possible, the Announcements category does not appear

![Screenshot 2021-05-30 at 11 32 38](https://user-images.githubusercontent.com/151693/120099230-c0181600-c13a-11eb-89f5-c9fe1cdf0325.png)

Am I missing something ?

I am the maintainer and admin of the repo

**Reply by evi-liu** [reply]
Hi @prudhomm, hm that's a strange bug. Are you still experiencing this behavior? We've pushed out quite a few fixes recently.

**Comment by kmgallahan** [comment]
@evi-liu Does the [current process for converting issues to discussions](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#converting-an-issue-to-a-discussion) also transfer all existing labels?

**Reply by evi-liu** [reply]
It does now! We recently shipped this ability :)
